src/runtime/source.c

```diff
@@ -145,7 +145,11 @@ video_source_init(struct video_source_s* self,
         .stream_id = stream_id,
         .to_filter = to_filter,
         .to_sink = to_sink,
-        .enable_filter = 0,
+        // TODO: this should be configured using the public API.
+        // Maybe if AcquireProperties::frame_average_count is non-zero,
+        // this should be true. Or we could use an enum to specify the
+        // filter, where the default value is none/off.
+        .enable_filter = 1,
```

Comment:
the intent was that this is initialized to zero, and then gets modified when frame_average_count is set. 

So `frame_average_count` is expected to be the public api. `enable_filter` is the corresponding state in the video source thread.

Reply:
Think I effectively did that in [my latest commit](https://github.com/acquire-project/acquire-video-runtime/pull/101/commits/ca199a8d16f72f3beac9643c3a10a6ce3dcb8f19), but I had to add an argument to `video_source_configure`.